p2panda-rs/src/permissions/key_group/views/key_group.rs

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use std::convert::TryFrom;
+
+use crate::document::{Document, DocumentId, DocumentViewId};
+use crate::operation::OperationValue;
+use crate::permissions::key_group::error::KeyGroupError;
+use crate::schema::system::SystemSchemaError;
+use crate::Validate;
+
+/// Represents a root key group definition.
+///
+/// Can be used to make a [`KeyGroup`].
+#[derive(Clone, Debug)]
+pub struct KeyGroupView(Document);
```

Comment:
I made this a thin wrapper, different to the schema definition view structs. I felt that document views are already so close to the data structure we want here that it's not warranted to make another struct.